docs/spark-connector/spark-catalog-hive.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Spark connector hive catalog"
+slug: /spark-connector/spark-catalog-hive
+keyword: spark connector hive catalog
+license: "Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2."
+---
+
+With the Gravitino Spark connector, accessing data or managing metadata in Hive catalogs becomes straightforward, enabling seamless federation queries across different Hive catalogs.
```

Comment:
Do you have some Hive or Iceberg specific configurations that should be listed here?

Reply:
no, the configurations are retrivied from Gravitino server.